pkgs/development/python-modules/flatbuffers/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv
+, buildPythonPackage
+, flatbuffers
+}:
+
+let
+  flatbuf = flatbuffers;
+in buildPythonPackage rec {
+  pname = "flatbuffers";
+  version = flatbuf.version;
+
+  src = flatbuf.src;
+  sourceRoot = "source/python";
+
+  pythonImportsCheck = [ "flatbuffers" ];
+
+  meta = {
+    description = "Python runtime library for use with the Flatbuffers serialization format";
+    homepage = flatbuf.meta.homepage;
+    license = flatbuf.meta.license;
+    maintainers = with stdenv.lib.maintainers; [ wulfsta ];
+  };
```

Comment:
personal preference
```suggestion
  meta = flatbuffers.meta // {
    description = "Python runtime library for use with the Flatbuffers serialization format";
    maintainers = with stdenv.lib.maintainers; [ wulfsta ];
  };
```

Reply:
Should be resolved in the latest commit.